README.rst

```diff
@@ -60,7 +60,7 @@ and are on a command prompt inside the LMS container.
 License
 -------
 
-The code in this repository is licensed under the Apache unless
+The code in this repository is licensed under the Apache v2 unless
```

Comment:
+"License", so that's it's an English sentence :)  (sorry to be a stickler).  Then it's good to go.